crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -259,6 +259,35 @@ static const uint8_t kAESCMACOutput[16] = {0xe7, 0x32, 0x43, 0xb4, 0xae, 0x79,
                                            0x08, 0x86, 0xe7, 0x9f, 0x0d, 0x3f,
                                            0x88, 0x3f, 0x1a, 0xfd};
 
+// AES-XTS test vector from
+// https://csrc.nist.gov/projects/cryptographic-algorithm-validation-program
```

Comment:
We can use the same test inputs in `CipherTestVector` if possible (`kAESKey_256`, `kPlaintext`, etc.). In my mind, the service indicator tests were mainly used for testing correctness of the service indicator. 
Using test vectors from Nist are nice though, we could add those to `xts_test.cc`?

Reply:
Good idea, I'll add some NIST test vectors to `xts_test.cc`.

It's probably also a good idea to use NIST KATs in `service_indicator_test.cc` as well, I'll open a new ticket for that.

Reply:
CryptoAlg-1265 for FIPS KATs in the service indicator.